<launch>
  <!-- phidgets_interface_kit interface_kit_node -->
<node pkg="phidgets_interface_kit" name="interface_kit_node" type="interface_kit_node" output="screen"/> 
<node pkg="distance_sensor" name="Compute_Velocity" type="compute_dist_vel" output="screen"/> 

</launch>